### ISSUE TITLE
Layout issue on the additive analysis on an IPhone 5S

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-ios#574]
![image](https://user-images.githubusercontent.com/1689815/73786049-d0afba80-4798-11ea-9a7f-4198c4f8b3cb.png)

**Comment by aleene** [comment]
The text should be multiline and scalable for accessibility.

**Comment by aleene** [comment]
The constraints might need some work. And the fonts should change to 
`UIFont(descriptor: UIFontDescriptor.preferredFontDescriptor(withTextStyle: UIFont.TextStyle.body), size: UIFontDescriptor.preferredFontDescriptor(withTextStyle: UIFont.TextStyle.body).pointSize)`

**Comment by aleene** [comment]
BLTNPageItem does not work on the iPad. It does not adapt itself to multi-application circumstances. It works only on full screen. In other circumstances it is off screen.

**Comment by teolemon** [comment]
![image](https://user-images.githubusercontent.com/1689815/75041291-95153000-54bc-11ea-9603-eccb3d7a1d24.png)
